contracts/lists/src/events.rs

```diff
@@ -0,0 +1,236 @@
+pub(crate) fn log_create_list_event(list: &ListExternal) {
+    env::log_str(
+        format!(
+            "{}{}",
+            EVENT_JSON_PREFIX,
+            json!({
+                "standard": "potlock",
+                "version": "1.0.0",
+                "event": "create_list",
+                "data": [
+                    {
+                        "list": list
+                    }
+                ]
+            })
+        )
+        .as_ref(),
+    );
+}
+
+/// update list event
+pub(crate) fn log_update_list_event(list: &ListInternal) {
+    env::log_str(
+        format!(
+            "{}{}",
+            EVENT_JSON_PREFIX,
+            json!({
+                "standard": "potlock",
+                "version": "1.0.0",
+                "event": "update_list",
+                "data": [
+                    {
+                        "list": list,
+                    }
+                ]
+            })
+        )
+        .as_ref(),
+    );
+}
+
+/// delete list event
+pub(crate) fn log_delete_list_event(list_id: ListId) {
+    env::log_str(
+        format!(
+            "{}{}",
+            EVENT_JSON_PREFIX,
+            json!({
+                "standard": "potlock",
+                "version": "1.0.0",
+                "event": "delete_list",
+                "data": [
+                    {
+                        "list_id": list_id,
+                    }
+                ]
+            })
+        )
+        .as_ref(),
+    );
+}
+
+/// upvote list event
+pub(crate) fn log_upvote_event(list_id: ListId, account_id: AccountId) {
+    env::log_str(
+        format!(
+            "{}{}",
+            EVENT_JSON_PREFIX,
+            json!({
+                "standard": "potlock",
+                "version": "1.0.0",
+                "event": "upvote",
+                "data": [
+                    {
+                        "list_id": list_id,
+                        "account_id": account_id,
+                    }
+                ]
+            })
+        )
+        .as_ref(),
+    );
+}
+
+/// remove upvote list event
+pub(crate) fn log_remove_upvote_event(list_id: ListId, account_id: AccountId) {
+    env::log_str(
+        format!(
+            "{}{}",
+            EVENT_JSON_PREFIX,
+            json!({
+                "standard": "potlock",
+                "version": "1.0.0",
+                "event": "remove_upvote",
+                "data": [
+                    {
+                        "list_id": list_id,
+                        "account_id": account_id,
+                    }
+                ]
+            })
+        )
+        .as_ref(),
+    );
+}
+
+/// update (add or remove) admins event
+pub(crate) fn log_update_admins_event(list_id: ListId, admins: Vec<AccountId>) {
+    env::log_str(
+        format!(
+            "{}{}",
+            EVENT_JSON_PREFIX,
+            json!({
+                "standard": "potlock",
+                "version": "1.0.0",
+                "event": "add_admins",
```

Comment:
event name `add_admins` should probably be `update_admin` since you can add or remove and emit the event

Reply:
good point, i'll update